docker/build.py

```diff
@@ -0,0 +1,115 @@
+# isort:block
```

Comment:
This is a lovely script overall. I think ideally these images will be built and deployed with releases via CICD which we can set up later.

Reply:
Yes! should we track it on an issue ? Could reference the documentation I'm adding on this PR on the manual release process

Reply:
Great call.